src/assets/main.css

```diff
@@ -1,35 +1,89 @@
 @import './base.css';
 
-#app {
-  max-width: 1280px;
-  margin: 0 auto;
-  padding: 2rem;
-  font-weight: normal;
+
+.board {
+  width: 860px;
+  height: 860px;
 }
 
-a,
-.green {
-  text-decoration: none;
-  color: hsla(160, 100%, 37%, 1);
-  transition: 0.4s;
-  padding: 3px;
+.board-row {
+  display: flex;
+  flex-wrap: nowrap;
 }
 
-@media (hover: hover) {
-  a:hover {
-    background-color: hsla(160, 100%, 37%, 0.2);
-  }
+.board-square {
+  height: 100px;
+  width: 100px;
+}
+
+.board-row-label {
+  width: 30px;
+  height: 100px;
+  line-height: 100px;
+  text-align: center;
 }
 
-@media (min-width: 1024px) {
-  body {
-    display: flex;
-    place-items: center;
+.board-column-label {
+  width: 100px;
+  height: 30px;
+  text-align: center;
+}
+
+.board-column-label:nth-child(1) {
+  margin-left: 30px
+}
+
+.board-square p {
+  text-align: center;
+  line-height: 100px;
+}
+
+.board-square-label,
+.board-square:hover .board-piece {
+  display: none;
+}
+
+.board-square:hover .board-square-label {
+  display: block;
+}
+
+.board-row:nth-child(odd) .board-square:nth-child(odd),
+.board-row:nth-child(even) .board-square:nth-child(even) {
+  background-color: #FFFFFF;
+  color: #000000;
+}
+
+.board-row:nth-child(odd) .board-square:nth-child(even),
+.board-row:nth-child(even) .board-square:nth-child(odd) {
+  background-color: #000000;
+  color: #FFFFFF;
+}
+
+
+
+@media (max-width: 1200px) {
+  .board {
+    width: 460px;
+    height: 460px;
```

Comment:
There are only 2 resolution sizes here. I added a task to the backlog to add more resolution sizes. Looks good for now!